tests/Zend/Captcha/ImageTest.php

```diff
@@ -399,6 +399,9 @@ public function testImageTagRenderedProperlyBasedUponDoctype()
 
 class Zend_Captcha_ImageTest_SessionContainer
 {
+    public $setExpirationHops;
+    public $setExpirationSeconds;
+    
```

Comment:
remove extra trailing space

Reply:
trailing space is still there

Reply:
Sorry take your time, i fixed here https://github.com/hungtrinh/zf1-future/commit/b357d445a567a8d574a532ca2fe316a6ca0558e9